<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>203</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="frame_range_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Frame Range:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="frame_range_start">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="frame_range_end">
    <property name="geometry">
     <rect>
      <x>241</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="file_name_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_name_input">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>50</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="export_button">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Export As Alembic</string>
    </property>
   </widget>
   <widget class="QLabel" name="root_label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Root:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="root_input">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>90</y>
      <width>231</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>406</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
